src/templates/ObservabilityPackDetails.js

```diff
@@ -32,6 +36,22 @@ const ObservabilityPackDetails = ({ data, location }) => {
       packId: pack.id,
     }
   );
+
+  const getPackUrl = () => {
```

Comment:
We might want to put this in it's own util file. I could see it being difficult to track down where this logic is, if we need to update it in the future. Maybe something like `utils/get-pack-url.js` - what do ya think?

Reply:
moved into its own file 👍